fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/pool/PoolConnectionProvider.java

```diff
@@ -275,4 +275,21 @@ public void describe(String prefix, StringBuilder cfg, String key) {
     public int getPoolSize() {
         return this.maxPoolSize;
     }
+
+    /**
+     * The caller is telling us they no longer need to use the pool so we can free
+     * any internal resources. This also let's us check for anything currently
+     * in-use that shouldn't be
+     * @throws IllegalStateException if there are open connections or a transaction
+     *         is active.
+     */
+    public void close() {
+        if (activeConnection.get() != null) {
+            throw new IllegalStateException("transaction still active");
+        }
```

Comment:
Fix the copyright at the top of the file

Reply:
fixed in next commit